src/assets/index.tsx

```diff
@@ -54,6 +54,7 @@ export const icons = {
   dataUsageIcon: require('./images/dataUsageIcon.png'),
   languageIcon: require('./images/languageIcon.png'),
   lockIcon: require('./images/lockIcon.png'),
+  lockIconGrey: require('./images/lockIconGrey.png'),
```

Comment:
A few PRs ago you added support for SVG icons. Maybe we should start using SVGs instead? Not saying to replace the old ones right now, but the new icons could be in SVG, right?

Reply:
using svg straight in view needed some more updates, but managed to get there :) It s working 🎊